client/packages/system/src/Item/api/api.ts

```diff
@@ -198,4 +202,9 @@ export const getItemQueries = (sdk: Sdk, storeId: string) => ({
 
     return result.updatePackVariant;
   },
+  deletePackVariant: async (input: VariantFragment) =>
+    await sdk.deletePackVariant({
+      storeId,
+      input: packVariantParses.toDelete(input),
```

Comment:
:nit perhaps that should be `packVariantParsers`? i.e. things that 'parse'

Reply:
Nice catch~ 🫣